### DISCUSSION TITLE
今天我的主号也登不上了

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#989] [General]
好在号还在，速成一下 kotlin 东山再起

**Comment by Arondight** [comment]
ipad 不是没问题嘛

**Reply by mark9804** [reply]
一直提示网络环境异常，之前是换了个密码登上去的，现在也整累了

这边的布局 bug 我会接着修的（如果还有的话），不过就只能盲修了

**Reply by Arondight** [reply]
啊这

**Comment by Arondight** [comment]
这玩意儿好像还行 https://github.com/koishijs/koishi

**Comment by mark9804** [comment]
![image](https://github.com/Arondight/Adachi-BOT/assets/9006264/b516cf9c-aa42-4700-9963-5b3ee964db67)

**Reply by Arondight** [reply]
哈哈哈现在是不是三方框架全噶了

**Reply by mark9804** [reply]
感觉是，解放了属于是

**Reply by Arondight** [reply]
哈哈哈太好了

**Comment by gaqwe767** [comment]
现在已经开始用签名api登录了，啥时候添加一下，8.8.88已经禁止登录45了...

**Reply by gaqwe767** [reply]
https://github.com/fuqiuluo/unidbg-fetch-qsign 用这个搭可以用

**Reply by mark9804** [reply]
这个有内存泄漏问题，经测试最高能漏12G

**Comment by mark9804** [comment]
如果要改的话 要改挺多东西的 难度不亚于重写（甚至比重写高）

我小组里开发者文档

https://doc.arona.diyigemt.com/install/login-bot#%E7%99%BB%E5%BD%95bot%E8%B4%A6%E5%8F%B7
